src/git-base.ts

```diff
@@ -31,19 +31,31 @@ export abstract class GitBaseCommand extends Command {
     this.bitBucket = await AxiosFactory.bitBucket()
   }
 
+  getJira(): AxiosJiraClient {
```

Comment:
I was thinking along the lines of an actual getter. Like: get jira() {

Reply:
works for me!